.copier-answers.yml

```diff
@@ -11,12 +11,12 @@ github_enable_makepot: false
 github_enable_stale_action: false
 github_enforce_dev_status_compatibility: false
 include_wkhtmltopdf: false
-odoo_test_flavor: Both
+odoo_test_flavor: OCB
 odoo_version: 16.0
 org_name: Odoo Community Association (OCA)
 org_slug: qrtl
 rebel_module_groups: []
-repo_description: null
+repo_description: OCA module
```

Comment:
```suggestion
repo_description: OCA Modules
```

Reply:
Thank you,Lin.
To maintain uniformity with custom/private repos, 
Nothing to fill out.
I'll come over change this right after I change ymd-oca.